controls/cf_execd.cf

```diff
@@ -4,6 +4,30 @@
 # These body settings determine the behaviour of cf-execd,
 # including scheduling times and output capture to
 # $(sys.workdir)/outputs and relay via email.
+#
+# If different schedules are needed based on class expressions then
+# an override for this file can be specified as an augment variable:
+# `cf_execd_override_policy`. The value is a path relative to `$(sys.inputs)`
```

Comment:
note: have to name it `body executor control`

Reply:
yes, but I will be closing this, recommend if customizations are needed, just change the vendored file. :+1: